server/controllers/groups.js

```diff
@@ -86,25 +86,25 @@ const getUserData = (groupId, tiers) => {
 };
 
 export const getUsers = (req, res, next) => {
-  let promise = getUserData(req.group.id, req.group.tiers);
-
-  if (req.query.filter && req.query.filter === 'active') {
-    const now = moment();
-    promise = promise.filter(backer => now.diff(moment(backer.lastDonation), 'days') <= 90);
-  }
+  auth.canEditGroup(req, res, (e, canEditGroup) => {
```

Comment:
Same comment here. `canEditGroup(user, group)` should be a function in a lib to be used by both auth and controllers.